pages/api/schools/collect.js

```diff
@@ -6,7 +6,7 @@ export default async (req, res) => {
   const session = await getSession({ req });
 
   // If session exists (thus, user is authenticated)
-  if (session && session.role === 'ADMIN') {
+  if (session && (session.role === 'ADMIN' || session.role === 'JUDGE')) {
```

Comment:
Without this the API returns a `401` for judge roles - we should return the data but filter it from the user on the FE

Reply:
Why do the judges need to be able to call api/schools/collect, they never need to see all the schools (just the ones associated with the appropriate performances).

Reply:
Looked into it, this is used to populate the filter options for the [performances page](https://github.com/uwblueprint/dancefest-web/blob/staging/pages/performances/index.js#L114)